tests/testsuite/standard_lib.rs

```diff
@@ -389,8 +389,29 @@ fn check_core() {
         .run();
 }
 
-#[cargo_test(build_std_mock)]
-fn test_std_on_unsupported_target() {
+#[cargo_test(build_std_mock, requires = "rustup")]
+fn build_std_with_no_arg_for_core_only_target() {
+    let has_rustup_aarch64_unknown_none = std::process::Command::new("rustup")
```

Comment:
I hate this. This is added to my list of cargo-test-macro overhaul.

Reply:
Overhaul `cargo-test-macro` or work on T-testing-devex to make it not needed anymore ;)

Reply:
We have a standard pattern for alternative targets in [`cargo_test_support::cross_compile`](https://docs.rs/cargo-test-support/latest/cargo_test_support/cross_compile/index.html), should we tie into that?

Reply:
That is actually a good idea, though I am not sure. The probe is based on building a simple cross-compile project. I don't want to disable all cross-compile test locally just because `aarch64-unknow-none` is missing.

Reply:
We could have a `cargo_test_support::build_std` with its own `disabled`.  More so I was wanting to highlight that pattern for consideration.

Reply:
Yeah. I'll keep this in mind. If we get any new tests with the same pattern we can extract the common part then.